policygenerator/policy-sets/community/ocp-best-practices/input-admin/policy-remove-kubeadmin.yaml

```
@@ -3,4 +3,4 @@ kind: Secret
 metadata:
   name: kubeadmin
   namespace: kube-system
-type: Opaque
+type: Opaque
```

nit: trailing newline
```suggestion
type: Opaque

```